src/parsers/espresso/parser.js

```diff
@@ -0,0 +1,271 @@
+import { ATOMIC_COORD_UNITS, coefficients } from "@exabyte-io/code.js/dist/constants";
+import { mix } from "mixwith";
+
+import { primitiveCell } from "../../cell/primitive_cell";
+import { Lattice } from "../../lattice/lattice";
+import math from "../../math";
+import { MaterialParser } from "../structure";
+import { FortranParserMixin } from "../utils/fortran";
+import { IBRAV_TO_LATTICE_TYPE_MAP, regex } from "./settings";
+
+export class ESPRESSOMaterialParser extends mix(MaterialParser).with(FortranParserMixin) {
+    parse(content) {
```

Comment:
I think ESLint rules should require a newline after class definition

Reply:
Not in my setup, which is pulled from /dev branch